src/_locales/en_US/messages.json

```diff
@@ -636,6 +644,10 @@
         "message": "Share",
         "description": "Tooltip that comes up when you hover over the share button in the upper right corner of the popup."
     },
+    "show_more_information": {
+        "message": "Click to show more information",
+        "description": "Tooltip that shows when you hover on the click-to-expand popup information sections."
```

Comment:
If we use a tooltip, it should show over the entire clickable header area, not just the carets.